src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/CreateSomaticPanelOfNormals.java

```diff
@@ -50,7 +44,7 @@
  *
  * <p>Step 1. Run Mutect2 in tumor-only mode for each normal sample.</p>
  * <pre>
- * gatk-launch --javaOptions "-Xmx4g" Mutect2 \
+ * gatk --javaOptions "-Xmx4g" Mutect2 \
```

Comment:
- `--germline_resource` --kebab-->
- Sorry for this reminder after you went through these already. But since we have to correct the kebab, remember to retain `--javaOptions "-Xmx4g"` if you deem it important to the tool functionality but otherwise remove. Also, I think `--javaOptions`--kebab-->`--java-options`.
- We should remove the `-L intervals.list` option from the M2 command as it is optional. I feel funny about starting the CSPON tooldoc with an M2 example command but having this really helps folks find what they need.

Reply:
done x 3